tools/rir_corpus_generator/conf/rir_corpus_v1.yaml

```diff
@@ -0,0 +1,54 @@
+output_dir: ${hydra:job.config_name}
+num_workers: 8
+random_seed: 42
```

Comment:
Some notes on arguments and their effect would be beneficial (either here or explicitly in documentation)